tabs/receiver.html

```diff
@@ -20,11 +20,11 @@
                 </div>
                 <div class="hybrid_element">
                     <input type="text" name="rcmap" spellcheck="false" /> 
-                    <select class="hybrid_helper"
-                        name="rcmap_helper">
-                        <option value="AETR">Default</option>
-                        <option value="AETR">Futaba / Hitec</option>
-                        <option value="TAER">JR / Spektrum / Graupner</option>
+                    <select class="hybrid_helper" name="rcmap_helper" id="rcmap_element">
+                        <!-- woga65: also map the AUX channels (will be overridden if the firmware sends only 4 channel's mapping) -->
+                        <option value="AETR1234">Default</option>
```

Comment:
If we are having to also have additional channels in the RC mapping. Please don't start an AUX section at 1. Just continue with the channel numbers.

BetaFlight uses the AUX1, AUX2... system and it is confusing, especially when new. Using AETR5678 makes more sense. People know what channel they have mapped their functions to on the radio. There's no need for them to have to remap their radio mixer in their head.

Reply:
Remember, "AETR5678" or "AETR1234" are channel names, not numbers. The channel number equals to the position of the letter from the left starting with one. 
So the wording for "TAER5687" would be: Throttle, Aileron, Elevator, Rudder, Channel five, Channel six, the channel that was previously channel seven, the channel that was previously channel eight.
(Just to explain the way I am thinking and why 5678 is more confusing to me. The requested change is on the way.)